web_monitoring/db.py

```diff
@@ -760,3 +781,25 @@ def get_version_by_versionista_id(self, versionista_id):
         # Make result look like the result of `get_version` rather than the
         # result of `list_versions`.
         return {'data': result['data'][0]}
+
+    def validate_db_credentials(self):
```

Comment:
Given this is a method on the DB client, we can probably drop `_db_` from the name here.

Reply:
Good call